test/HyperliquidInterchainToken.js

```diff
@@ -12,6 +12,9 @@ const { getRandomBytes32, getEVMVersion } = require('./utils');
 const { deployAll } = require('../scripts/deploy');
 const { ITS_HUB_ADDRESS } = require('./constants');
 
+// Precalculated storage slot: bytes32(uint256(keccak256('hyperliquid-interchain-token-deployer')) - 1)
+const CURRENT_DEPLOYER_SLOT = '0x8f802fa116fa5e7a7e0f94874e9214762d5a91faa240ab5140b3a79fbca28666';
```

Comment:
avoid hardcoded magic constants. Compute the actual keccak256 hash in the test and query that slot. The only reason we did this in Solidity was due to an optimization for complex constant expressions